java/ql/lib/ext/hudson.util.jna.model.yml

```diff
@@ -0,0 +1,8 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: sinkModel
+    data:
+      - ["hudson.util.jna", "GNUCLibrary", True, "open", "(String,int)", "", "Argument[0]", "read-file", "ai-generated"]
+# suggested label is not supported:      - ["hudson.util.jna", "Kernel32", True, "MoveFileExA", "(String,String,int)", "", "Argument[0]", "delete-file", "ai-generated"]
```

Comment:
```suggestion
      - ["hudson.util.jna", "Kernel32", True, "MoveFileExA", "(String,String,int)", "", "Argument[0]", "delete-file", "ai-generated"] # should be delete-file
```

Reply:
🙈 This suggestion was incomplete -- it left the `delete-file` label in place. I also didn't spot this. Fixed now.

Reply:
Sorry 🤦. Thanks for fixing it!